source/development/feature-flags.html.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Feature flags in GOV.UK Frontend
+weight: 1
+last_reviewed_on: 2023-11-28
+review_in: 6 months
+---
+# Using feature flags in GOV.UK Frontend
+
+Feature flags are a way for us to release large scale or experimental work early by allowing users to opt in our out of a new, breaking change or feature. Use this guidance to decide if a feature flag is useful for your use case.
+
+## When a feature flag is useful
+Feature flags are useful when the following conditions are true.
+
+### The thing is a breaking change
+Feature flags should only be used for things considered breaking changes. This is something where users will potentially need to change their service, or assess if their service needs changing, in order to continue staying up to date with govuk-frontend. In this case, a feature flag would be a way for a user to stay up to date whilst also opting out of the feature for a period of time. We don’t need to do this for minor releases or new features such as a new component because a user can opt out by simply not using the new feature.
```

Comment:
```suggestion
You should only use Feature flags for things considered breaking changes. Assess whether a breaking change will cause a need for users to amend their service in order to stay up to date with govuk-frontend. In this case, a feature flag would be a way for users to stay up to date while also being able to opt out of the feature for a period of time. Feature flags are not needed for minor releases or new features, such as a new component, because it is simpler to opt out by not using the new feature.
```

Reply:
I think I've changed the meaning of this. I was trying to clarify who was doing what action.

Reply:
I think the only issue with the suggestion is the 2nd sentence. I was trying to clarify what a breaking change is to make it clear why a feature flag is only necessary for breaking changes. I do that at the end with the new feature explanation though. What do you think about me just removing that 2nd sentence?

Reply:
I see. I've nicked the breaking change definition from the frontend site as a replacement in the second line. Does it work in this context?

Reply:
It does! Accepting.